ci/launch_cluster.sh

```diff
@@ -33,24 +46,36 @@ wget 'https://downloads.dcos.io/dcos-test-utils/bin/linux/dcos-launch' && chmod
 envsubst <<EOF > config.yaml
 ---
 launch_config_version: 1
-template_url: $TEMPLATE
 deployment_name: $DEPLOYMENT_NAME
-provider: aws
+platform: aws
+instance_type: m4.xlarge
+os_name: cent-os-7-dcos-prereqs
 aws_region: us-west-2
+aws_key_name: default
+provider: onprem
+installer_url: $INSTALLER
 key_helper: true
-template_parameters:
-    DefaultInstanceType: m4.large
-    AdminLocation: 0.0.0.0/0
-    PublicSlaveInstanceCount: 1
-    SlaveInstanceCount: 3
-    LicenseKey: $DCOS_LICENSE
+num_masters: 3
+fault_domain_helper:
+  LocalZone:
+    num_zones: 2
+    num_public_agents: 1
+    num_private_agents: 2
+    local: true
+  RemoteZone:
+    num_zones: 3
```

Comment:
Could we go with two zones instead?

Reply:
We could give it a try. It was just a hunch that 2 might not show some corner-case errors due to fewer choices that could be accidentally picked.